qa/L0_server_launch_modes/test.sh

```diff
@@ -156,6 +156,9 @@ run_server_launch_modes "$CURRENT_GPUS"
 CURRENT_GPUS=${GPUS[@]:1}
 run_server_launch_modes "$CURRENT_GPUS"
 
+CURRENT_GPUS=""
+run_server_launch_modes "$CURRENT_GPUS"
```

Comment:
I think this should trigger. 
It is because the `gpus` array is passed to the `run_server_launch_modes` function, and then converted to the `--gpus` flag for the model analyzer by the `convert_gpu_array_to_flag` function. It will result in the `run_server_launch_modes` function launching the model analyzer with an empty `--gpus` flag for different launch modes.

Reply:
I don't think it does this. https://github.com/triton-inference-server/model_analyzer/blob/main/qa/L0_server_launch_modes/test.sh#L60 If the "CURRENT_GPUS" is not specified it will remove the `--gpus` flag altogether.

Reply:
Oh no. This is fixed, with a special case for empty flag